tests/integration/test_error_handling.py

```
@@ -0,0 +1,659 @@
+provider = "aws"
+region = "us-east-1"
+
+[[vault.sync.mappings]]
+secret_name = "test/secret"
+folder_path = "."
+environment = "production"
+"""
+        (work_dir / "envdrift.toml").write_text(config_content)
+        (work_dir / ".env.production").write_text('SECRET="encrypted:..."')
+
+        result = subprocess.run(
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)